test/CommonSpaces/unit_common_spaces.jl

```diff
@@ -34,7 +37,7 @@ using Test
         h_elem = 10,
         n_quad_points = 4,
         horizontal_layout_type = DataLayouts.IJHF,
-        staggering = Grids.CellCenter(),
+        staggering = CommonSpaces.CellCenter(),
```

Comment:
Can you please revert this? `CellCenter`/`CellFace` is not an object defined in `CommonSpaces`.

Reply:
I thought we decided to re-export it and use it directly from `CommonSpaces` when working with `CommonSpaces`. 

This change ensures that those looking at this code have a good pattern to copy.

Reply:
I'm fine with re-exporting it, but that means that users shouldn't have to qualify it. That's not where it comes from, I think it makes the most sense to qualify objects from their origin, not from wrappers. For example, we did this a lot when stripping out dependencies in ClimaTimesteppers when moving from DiffEqBase to SciMLBase, which is where the methods originate from.

Reply:
I removed the qualifier.

Reply:
Thank you!